internals/osutil/user.go

```diff
@@ -56,6 +57,12 @@ func RealUser() (*user.User, error) {
 	if _, ok := err.(user.UnknownUserError); ok {
 		return cur, nil
 	}
+	// Workaround for https://github.com/golang/go/issues/67912, until our
+	// minimum Go version has a fix for that. In short, user.Lookup sometimes
+	// doesn't return UnknownUserError when it should.
+	if err != nil && strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
Is a “C” or an en_* locale guaranteed for pebble?

I’m assuming that go localise strerror, tbh I don’t know if it does.

Reply:
@benhoyt can you please replace all the string literals here with `syscall.Errno(2).Error()`

Reply:
@dimaqq Yep, Go doesn't use the C locale environment variables (thankfully!). It's a hard-coded string, see https://github.com/golang/go/blob/beaf7f3282c2548267d3c894417cc4ecacc5d575/src/syscall/syscall_unix.go#L110 and https://github.com/golang/go/blob/beaf7f3282c2548267d3c894417cc4ecacc5d575/src/syscall/zerrors_linux_amd64.go#L1361

@hpidcock Good call. One better: `syscall.ENOENT.Error()`. :-)